policy_packs/gcp/network/enforce_default_vpc_network_is_not_used_for_projects/README.md

```diff
@@ -3,7 +3,7 @@ categories: ["networking", "security"]
 primary_category: "networking"
 ---
 
-# Enforce Default VPC Network to Not Be Used Within GCP Projects
+# Enforce Default VPC Network Is Not Used in GCP Projects
```

Comment:
Please check if the Policy Pack AKA matches the title.

Reply:
Updated 👍